README.md

```diff
@@ -8,20 +8,138 @@ Please sign up through [this form](https://forms.gle/1w8umSbTF4JTPwyj9) if you h
 
 **Can't wait to have you officially join the team!**
 
-### Pick Your Article
+## How to post-edit an article
 
-Once you're in as a Collaborator, it's time to choose which article you'd like to work on. Check out the list of available issues/articles and pick the one that speaks to you from the Project of the spoken language you would like to help on.
+Once you're in as a Collaborator, it's time to choose which article you'd like to work on.
 
-### Claim Your Article
+In this repository, you will [post-edit](https://en.wikipedia.org/wiki/Postediting) the machine translations.
 
-Found the perfect piece to translate? Awesome! Just comment `/translate` on the issue. This will automatically assign the issue to you and move the card to the "in Translation" status.
+There are machine-translated markdown files. Carefully revise it and make any corrections or improvements. Then, open a PR to propose the changes. The language lead will review your PR and publish it in our publication.
+
+Here are the detailed steps.
+
+### Select the article to work on
+
+1. Select an article from the list of articles to be post-edited. 
+
+Check out the list of available articles in the GitHub Projects of your language, and pick the issue that you'd like to work on.
+
+2. Comment `/postedit` on the issue. This will automatically assign the issue to you and move the card to the "in Post-edit" status.
```

Comment:
Just saw from the Japanese board that there's a `in Postediting` status instead of `in Postedit`. Does it work both ways or do we need to fix it?